source/blender/gpu/shaders/gpu_shader_vertex_world.glsl

```diff
@@ -79,7 +81,14 @@ void main()
 	varposition = gl_Vertex.xyz;
 
 	varnormal = normalize(-varposition);
+
+#ifdef CLIP_WORKAROUND
 	// Always set clip distance to 1 to disable clipping.
 	for (int i = 0; i < 6; ++i) {
 		gl_ClipDistance[i] = 1.0;
```

Comment:
This is to disable clipping for world (background rendering) did you test a scene with planar reflection and a complex world ?

Reply:
Yes, I just tested it and it works wrong.
![image](https://user-images.githubusercontent.com/1015605/50305328-c5190b80-0492-11e9-8ab6-686c0fe56db2.png)
I'm looking for a solution.

Reply:
Fix commited
![image](https://user-images.githubusercontent.com/1015605/50305777-32796c00-0494-11e9-8e7f-1bbce9fcb761.png)